src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -889,3 +870,66 @@ Now your Slack integration is available to all projects in your Sentry organizat
 #### Deleting the legacy Slack integration
 
 Once you configure the global Slack integration and Alert Rules, you can disable the old Slack integration. You’ll need to go to each project that has it enabled and disable it. We recommend disabling the legacy integration after setting up the global integration.
+
+
+
+## Other
```

Comment:
What does "Other" mean? Can we give this a more descriptive heading?

Reply:
@MimiDumpling Other means it doesn't fit into the other categories. Should I say that?